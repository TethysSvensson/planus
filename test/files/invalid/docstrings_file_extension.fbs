//! Okay
/// Bad1
//! Bad2
/// Bad3
file_extension /// Bad4
/// Bad5
"foo" /// Bad6
/// Bad7
; /// Bad8
/// Bad9
//! Bad10
