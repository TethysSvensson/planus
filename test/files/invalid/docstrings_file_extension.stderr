error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstrings_file_extension.fbs:2:1
  │
2 │ /// Bad1
  │ ^^^^^^^^

error: Inner doc comments are not meaningful here.
  ┌─ test/files/invalid/docstrings_file_extension.fbs:3:1
  │
3 │ //! Bad2
  │ ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstrings_file_extension.fbs:4:1
  │
4 │ /// Bad3
  │ ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstrings_file_extension.fbs:5:16
  │
5 │ file_extension /// Bad4
  │                ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstrings_file_extension.fbs:6:1
  │
6 │ /// Bad5
  │ ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstrings_file_extension.fbs:7:7
  │
7 │ "foo" /// Bad6
  │       ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstrings_file_extension.fbs:8:1
  │
8 │ /// Bad7
  │ ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstrings_file_extension.fbs:9:3
  │
9 │ ; /// Bad8
  │   ^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstrings_file_extension.fbs:10:1
   │
10 │ /// Bad9
   │ ^^^^^^^^

error: Inner doc comments are not meaningful here.
   ┌─ test/files/invalid/docstrings_file_extension.fbs:11:1
   │
11 │ //! Bad10
   │ ^^^^^^^^^

Error: Bailing because of errors
